PUIR_304b/common/cb_types.txt

```diff
@@ -755,7 +755,44 @@ cut_down_to_size = {
 	}
 }
 
+# Cut Down to size (Reperations)
+cut_down_to_size = {

```

Comment:
Just wanted to make it work functionally as fast as possible, Most text is the same as normal C_D_T_S, but can / will ofcourse be changed when some one bothers to do it :)